README.md

```diff
@@ -13,3 +13,96 @@ Please see the [contributing guide](https://github.com/exercism/docs/tree/master
+
+## Contributing a New Exercise ##
+
+- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+
+- Please make sure the new exercise conforms to specifications in the [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repo.  (Note if slight changes are needed for Rust
+specific issues, see `.meta\hints.md below.)
+
+- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+
+- Exercises should use only the Rust core libraries.  If you must use an external crate, see note below about `Cargo-example.toml`.  `Cargo.toml` should not have external dependencies as we don't want to make the user assume which crates are required.
+
+-  !!!!! Not sure if this is accurate !!!!!  Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
+[exercism/problem-specifications](https://github.com/exercism/problem-specifications) repository. Further explanation can be found in [fixing-exercise-readmes](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/exercise-readmes.md)
+
+- Each exercise should have:
+---
+    exercises/exercise-name/
+                            tests/exercise-name.rs  <- a test suite
+                            src/lib.rs              <- an empty file or with exercise stubs
+                            example.rs              <- example solution that satisfies tests
+                            Cargo.toml              <- with version equal to exercise defintion
+---
+
+- An exercise may contain `.meta/hints.md`.  This is optional and will appear after the normal exercise
+  instructions if present.  Rust is different in may ways from other languages.  This is a place where 
+  the differences required for Rust are explained.  If it a large change, you may want to call this out
+  as a comment at the top of `src/lib.rs`, so the user recognises to read this section before starting.
+
+- If an `example.rs` uses external crates, include `Cargo-example.toml` so that `_tests\check-exercises.sh` can compile with these when testing.
+
+- We are currently committing Cargo.lock and README.md.  This may change.
+
+- Be sure to add it to the appropriate place in the `config.json` file.  Position in the file determines the order exercises are sent.   Generate a unique UUID for the exercise.  Current difficuly levels in use are 1, 4, 7 and 10.
+
+- !!!!! Not sure if this is accurate !!!!!  Also, please run `bin/fetch-configlet && bin/configlet` to ensure the exercise is configured correctly.
```

Comment:
Eh... this sentence is accurate in that it is possible to do so. I'm fine with calling this an optional step, since Travis CI will do it anyway.